translate/commands.yaml

```diff
@@ -1,8 +1,3 @@
 commands:
   translate:
-    description: Translates text to a given language
-    parameters:
-      - name: text
-        optional: false
-      - name: language
-        optional: true
+    description: Translates text to a given language
```

Comment:
Your implementation is interesting in that it parses the full command text and uses the first token as the language. I like that in that it could actually make the first token optional (if not recognized as a target language).

To keep the parameters, the first `language` stays and is now `required.
The second is a `vararg` and would be specified as 
```
   parameters:
      - name: language
         optional: false
      - name: "..."
```

Soliciting feedback cc @bhageena @satyarohith, thoughts ?

Reply:
I wish we had support for options with `varargs` but we cannot use anything optional (options, parameters) with `varargs` yet, so this looks like the way to go.